.github/workflows/all_plugins.yaml

```diff
@@ -21,13 +21,10 @@ jobs:
       - name: "Install Tools"
         run: |
           ./.github/workflows/scripts/install-tools.sh
-          flutter --suppress-analytics pub global activate tuneup
       - name: "Bootstrap Workspace"
         run: melos bootstrap
-      - name: "Dart Analyze"
-        run: |
-          melos exec -c 3 -- \
-            tuneup check
+      - name: "Run Analyze"
+        run: melos run analyze --no-select
```

Comment:
What's the `--no-select`?

Reply:
Doh, not needed on this one so will remove.

But to explain what it is, any script with a `select-package` configuration in the definition in melos.yaml e.g. https://github.com/FirebaseExtended/flutterfire/blob/master/melos.yaml#L31-L36 will prompt the user to select a package to run it against.

e.g. I can run `melos run build:example_android` and by default it'll ask me to choose from a pre-filtered list of packages in the workspace to run the actual script against. I could even just do `melos run` and will be prompted to select a script, then if the script has a select package, a package.

`--no-select` disables the selection behaviour but keeps the filters, meaning it'd run against all packages matching the filter.